pkgs/misc/emulators/libdsk/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+
+  pname = "libdsk";
+  name = "libdsk-${version}";
+  version = "1.5.8";
+
+  src = fetchurl {
+    url = "http://www.seasip.info/Unix/LibDsk/${name}.tar.gz";
+    sha256 = "1fdypk6gjkb4i2ghnbn3va50y69pdym51jx3iz9jns4636z4sfqd";
+  };
+
+  meta = with stdenv.lib; {
+    description = "a library for accessing discs and disc image files";
+    homepage = http://www.seasip.info/Unix/LibDsk/ ;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.genesis ];
```

Comment:
It's generally preferred to use a maintainer handle that matches the github handle.

Reply:
i will change maintainers in a separate PR for all my derivation.